### ISSUE TITLE
Moon position

### ISSUE THREAD

**Issue by raidernick** [KSP-RO/RealSolarSystem#171]
@PhineasFreak the moon is not in the correct position on relevant dates. It is either set to start at an incorrect position when you first create the save or it is drifting over time. In the 1960s to do any of the lunar missions, the moon is never where it should be on the time and date when the actual mission took place.

**Comment by PhineasFreak** [comment]
That cannot be fixed without Principia. The stock "integrator" is far too simplified when calculating the future positions of celestial bodies and, of course, does not account for any other gravitational effects. It **will** drift in a couple of years from the start date (1950-01-01), as i had the chance to test so by doing recreations of the early Pioneer direct ascent missions.

TL;DR not fixable in stock.

**Comment by raidernick** [comment]
then it should be set to drift to the correct position by 1969, it doesn't matter if it starts in the right spot as moon missions in 1951 didn't exist.

**Comment by PhineasFreak** [comment]
And then you are going to have the Moon being on a different position at the start of a game. Or after 1969.

**Edit 1:** the Apollo program was not the only lunar program, why should it matter so much? If someone wants "Apollo 50" then we could:

* Create a save file that has been hand-edited to have the Moon at the proper phase angle (for those that do not like Principia)
* Suggest the use of Principia

**Edit 2:** and, of course, accuracy without Principia does not only affect the Earth-Moon system: https://github.com/KSP-RO/RealSolarSystem/issues/88

**Comment by raidernick** [comment]
well unfortunately it has nothing to do with liking or not liking principia, with all the rest of the mods to still get performance something has to be sacrificed somewhere, and that is the best place. It's not necessary to play and saves a large amount of performance by not using it. Regardless, there are other moon programs yes but the most important ones took place in the 60s. It should take several years to drift any significant amount so it could feasibly be set to be in more or less the right place in the 60s in general.

**Comment by eggrobin** [comment]
> It should take several years to drift any significant amount so it could feasibly be set to be in more or less the right place in the 60s in general

Note the length of the [precession cycles](https://en.wikipedia.org/wiki/Lunar_precession):
- 18.6 years for the nodal precession;
- 8.85 years for the apsidal precession;

That means that:
- 9.3 years after being correct with stock orbits, the orbital plane of the moon is maximally wrong;
- 4.4 years after being correct with stock orbits, the position of the lunar perigee along the orbit (the argument of periapsis) is maximally wrong.

Both of these durations are under a decade.

For comparison: there are 3.43 years between the Apollo 11 launch and the Apollo 17 splashdown, so if both are correct for Apollo 11, by the end of the Apollo programme:
- the lunar perigee is 139° out of phase;
- the ascending node of the orbit of the Moon (with respect to the ecliptic) is 67° out of phase.

This is a very significant drift already, and we are only talking about the handful of years of the Apollo programme; consider something spanning a decade, like Луна-2 through Луна-17/Луноход-1, and no choice is good.

**Comment by eggrobin** [comment]
There is one thing that could be improved, namely phases.
For now, the stock RSS moon has a period (which I believe is the osculating period at the initial time) of 27.44 days; if you want phases that stay roughly correct in the long run, you could use the [mean *sidereal* period of the Moon](https://en.wikipedia.org/wiki/Orbit_of_the_Moon#Lunar_periods).

With such a change, moon phases will not drift; of course, the other issues will remain:
- the perigee-to-perigee time (anomalistic period) of the Moon will be wrong;
- the node-to-node time (nodal or draconic period) of the Moon will be wrong;
- the orbital plane of the moon will be wrong (with quickly-varying wrongness, see the preceding post);
- the position of the perigee along the orbit will be wrong (with extremely-quickly-varying wrongness).

In particular the unavoidable wrongness of the orbital plane has a strong influence on gameplay, since it means that both the inclination with respect to the equator, and the node with respect to the equator, (affecting, respectively, launch site usability/need for a dogleg, and launch window timing) are very wrong most of the time.
